apriltags.orogen

```diff
@@ -46,10 +46,11 @@ task_context "Task" do
     #Output Ports
     output_port("output_image", ro_ptr("/base/samples/frame/Frame"))
         .doc 'Image that shows the detected markers'
-#    output_port("marker_poses", "std/vector<base::samples::RigidBodyState>")
-#        .doc 'It contains the calculated marker poses in respect to camera frame of all visible markers'
-    output_port("marker_poses", "base::samples::RigidBodyState")
+    output_port("marker_poses", "std/vector<base::samples::RigidBodyState>")
         .doc 'It contains the calculated marker poses in respect to camera frame of all visible markers'
+#    output_port("marker_poses", "base::samples::RigidBodyState")
+#        .doc 'It contains the calculated marker poses in respect to camera frame of all visible markers'
+    output_port("marker2cam", "base::samples::RigidBodyState")
```

Comment:
Please add a little description of this output port and you can delete the commented lines.